### DISCUSSION TITLE
Using right shift and modulus operators

### DISCUSSION THREAD

**Discussion by Kyzegs** [prisma/prisma#17501] [Q&A]
Hey there! I have a SQL (MySQL) query that looks like this:

```sql
SELECT * FROM guilds WHERE (id >> 22) % 3 in (1, 2);
```

Is it possible to do queries like this with Prisma? I was unable to find anything about it.

**Comment by nurul3101** [comment]
Hey @Kyzegs 👋 

You would need to use [Raw Queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) to use right shift and modulus operators. They aren't natively supported in prisma yet.

**Reply by Kyzegs** [reply]
Are there plans to add these?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
